src/pages/Dashboard/Modals/BorrowRepay/AccountData/index.tsx

```diff
@@ -0,0 +1,170 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { FormikProps } from 'formik';
+import BigNumber from 'bignumber.js';
+
+import { SAFE_BORROW_LIMIT_PERCENTAGE } from 'config';
+import { Asset } from 'types';
+import { AuthContext } from 'context/AuthContext';
+import { FormValues } from 'containers/AmountForm';
+import { useUserMarketInfo } from 'clients/api';
+import { formatToReadablePercentage } from 'utilities/common';
+import calculateDailyEarningsCentsUtil from 'utilities/calculateDailyEarningsCents';
+import { calculateYearlyEarningsForAssets } from 'utilities/calculateYearlyEarnings';
+import calculatePercentage from 'utilities/calculatePercentage';
+import {
+  BorrowBalanceAccountHealth,
+  LabeledInlineContent,
+  ValueUpdate,
+  Delimiter,
+} from 'components';
+import { useTranslation } from 'translation';
+import { useStyles } from '../../styles';
+
+export interface IAccountDataProps {
+  asset: Asset;
+  amount: FormikProps<FormValues>['values']['amount'];
+}
+
+const AccountData: React.FC<IAccountDataProps> = ({ asset, amount }) => {
+  const { t } = useTranslation();
+  const styles = useStyles();
+  const { account } = React.useContext(AuthContext);
```

Comment:
If we wanted to save this call the `accountAddress` and `userTotals` are passed down from the Dashboard container.

Reply:
If we want to do that we can also refactor the Supply Modal

Reply:
I'd rather import the account from here. We're aren't doing any additional HTTPS request (thanks to the cache) or anything that should have a significant performance impact, but this way we don't have drill down props.

Also, I wouldn't refactor the Supply modal since it's not creating any issue right now. In the case of the Borrow modal, both tabs are too different to be merged into one component, so I hoped for merging the elements that could be merged together instead.